broker/src/main/java/io/camunda/zeebe/broker/system/monitoring/BrokerHealthCheckService.java

```diff
@@ -100,19 +101,21 @@
   private volatile boolean brokerStarted = false;
   private final HealthMonitor healthMonitor;
   private final MemberId nodeId;
-  private final PartitionGroup partitionGroup;
 
-  public BrokerHealthCheckService(
-      final BrokerInfo localBroker, final PartitionGroup partitionGroup) {
-    this.partitionGroup = partitionGroup;
+  public BrokerHealthCheckService(final BrokerInfo localBroker) {
```

Comment:
:question: IIRC, the service knew that it was not ready because it was waiting on some partition information. So now to use it, you first have to register the partition manager at which point its status can then be changed? Is it statically not ready until then?

Reply:
Didn't get this comment. What is your concern here?

Reply:
Not a concern, just a question :sweat_smile: I'm trying to understand the semantics of the class now. So let me rephrase - what's the health status of the service _before_ I register the partition manager?

Reply:
Good question. Let me write a test for it

Reply:
the health status, ready status and started status are all false before the partition manager is registered.